### PR TITLE
Add time_minutes runtime_attribute

### PR DESCRIPTION
**Pull request by DavyCats**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] `parameter_meta` for each task is up to date.

### CODE REVIEW

File: star.wdl [hunk 1]
```diff
@@ -102,6 +102,7 @@ task Star {
 
         Int runThreadN = 4
         String memory = "48G"
```
Comment (rhpvorderman) [inline star.wdl:104]: I have been running star lately. The memory usage is roughly equal (it seems to be slightly lower based on MaxRSS) than the index size. May we can do `ceil(size(indexFiles, "G") * 1.2)`  as a memory estimate?

File: stringtie.wdl [hunk 2]
```diff
@@ -33,6 +33,7 @@ task Stringtie {
 
         Int threads = 1
         String memory = "10G"
```
Comment (rhpvorderman) [inline stringtie.wdl:35]: Does it need that much? I have checked a 400ish MB BAM file and usage seems to be similar. But there is also a 1.3GB gtf involved.
Hmm. This is undecided.
Can we put memory at a static 2G for now? From MaxVMSize seems to be 5x the MaxRSS number. So I think we can safely lower memory by 5X. 
If we run into issues we can look at setting it dynamically.

File: vardict.wdl [hunk 3]
```diff
@@ -48,14 +48,15 @@ task VarDict {
         Float minimumAlleleFrequency = 0.02
 
         Int threads = 1
-        String memory = "40G"
+        String memory = "18G"
```
Comment (rhpvorderman) [inline vardict.wdl:51]: 17G

File: biopet/biopet.wdl [hunk 4]
```diff
@@ -264,8 +269,9 @@ task ScatterRegions {
         File? bamFile
         File? bamIndex
 
-        String memory = "24G"
-        String javaXmx = "8G"
+        String memory = "1G"
+        String javaXmx = "500M"
```
[the remark this answers is not in the document]
Comment (DavyCats) [reply biopet/biopet.wdl:273]: Tested this with RNAseq and had no problems, shouldn't be any different with DNA

File: biopet/sampleconfig.wdl [hunk 5]
```diff
@@ -34,7 +34,7 @@ task SampleConfig {
         String? jsonOutputPath
         String? tsvOutputPath
 
-        String memory = "8G"
+        String memory = "18G"
```
[the remark this answers is not in the document]
Comment (DavyCats) [reply biopet/sampleconfig.wdl:37]: Not in any of the up to date pipelines 😛

File: htseq.wdl [hunk 6]
```diff
@@ -33,6 +33,7 @@ task HTSeqCount {
         Array[String] additionalAttributes = []
 
         String memory = "40G"
```
[the remark this answers is not in the document]
Comment (DavyCats) [reply htseq.wdl:35]: Uhm, I believe the memory requirements are usually lower, but can jump up quite a bit for some samples. @JasperBoom was at one point trying to use name sorted BAMs to solve that, but that was put on hold.

[the remark this answers is not in the document]
Comment (rhpvorderman) [reply htseq.wdl:35]: Oh yes, I remember.

File: vardict.wdl (hunk 3 above)
Comment (DavyCats) [reply vardict.wdl:51]: vardict may need some extra memory overhead, since there are some pipes involved

**Pull request merged**